ui/stash.go

```diff
@@ -64,6 +64,14 @@ const (
 	stashStateShowingError
 )
 
+func (s stashViewState) String() string {
```

Comment:
Are these `String` functions actually used somewhere yet? I think we may be able to use them to clean up the code a bit.

Reply:
I use them in debugging (in fact I used some of them to debug this). I had taken them out at one point, but since I’ve added them back I figured I’d keep them around. Been back and forth about them, though, and they do have the potential for rot, which is an argument to remove them.

Reply:
Alright, dropped 'em. Let's keep this code lean.